handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,82 @@
+# MyProductArea Direction
+
+This page outlines the vision, strategy and goals of the MyProductArea team. You can use this section to describe what your scope is, and where your product is today.
```

Comment:
What do you think about the following? I'm not sure if this philosophy is controversial if not, but if so I could propose it in a separate PR.

```suggestion
This page outlines the vision, strategy and goals of the MyProductArea team. You can use this section to describe what your scope is, and where your product is today.

As you write this document, keep in mind that focusing on problems to solve more than lists of features tends to be a better way to approach communicating with customers, as it leaves room for learning between what you know now and what you deliver in the end. This is not to say you never talk about features, but especially as you get [further out in planning](https://medium.com/@jobv/the-nearsighted-roadmap-57fa57b5906a) the blend of features vs. problems to solve should move more towards the latter.
```

Reply:
I don't think it is controversial, Christina repeats this over and over again to explain how our team works to other teams. Adding.